chatterbox_onnx.py

```
@@ -257,11 +242,40 @@ def _generate_waveform(
+    def _watermark_and_save(self, wav, output_file_name: str, apply_watermark=False):
+
+        # 3. Optional: Apply Watermark
+        if apply_watermark:
+            print("Applying audio watermark...")
+            try:
+                import perth
+                watermarker = perth.PerthImplicitWatermarker()
+                wav = watermarker.apply_watermark(wav, sample_rate=S3GEN_SR)
+            except ImportError:
+                print("Warning: 'resemble-perth' not installed. Watermark skipped.")
+            except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)